src/effects/Distortion.h

```diff
@@ -90,30 +95,24 @@ class EffectDistortion final : public StatefulPerTrackEffect
       const override;
    OptionalMessage DoLoadFactoryPreset(int id);
 
-   unsigned GetAudioInCount() const override;
-   unsigned GetAudioOutCount() const override;
-   bool ProcessInitialize(EffectSettings &settings, double sampleRate,
-      ChannelNames chanMap) override;
-   size_t ProcessBlock(EffectSettings &settings,
-      const float *const *inBlock, float *const *outBlock, size_t blockLen)
-      override;
-   bool RealtimeInitialize(EffectSettings &settings, double sampleRate)
-      override;
-   bool RealtimeAddProcessor(EffectSettings& settings, EffectOutputs *pOutputs,
-      unsigned numChannels, float sampleRate) override;
-   bool RealtimeFinalize(EffectSettings &settings) noexcept override;
-   size_t RealtimeProcess(size_t group,  EffectSettings &settings,
-      const float *const *inbuf, float *const *outbuf, size_t numSamples)
-      override;
+   // Need this dummy method, else it won't build
```

Comment:
But at this commit, as with BassTreble, you can change the effect base class to `PerTrackEffect` and also change the #include above.  Then this is not needed